From 48b2521b1894e09248a6e099c0cb1783015f66a8 Mon Sep 17 00:00:00 2001
From: Henry Gebhardt <hsggebhardt@googlemail.com>
Date: Thu, 25 Nov 2010 22:58:43 +0100
Subject: [PATCH] Revert "Revert "Revert "fsck: add new -l switch to fsck mount options"""

This reverts commit e9ecea880571e520c60fae2dd2f10fcfd7dd1ab8.

Needs development version of util-linux.
---
 src/fsck.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/src/fsck.c b/src/fsck.c
index fdcf6b4..dfaf221 100644
--- a/src/fsck.c
+++ b/src/fsck.c
@@ -142,7 +142,7 @@ static void test_files(void) {
 }
 
 int main(int argc, char *argv[]) {
-        const char *cmdline[8];
+        const char *cmdline[7];
         int i = 0, r = EXIT_FAILURE, q;
         pid_t pid;
         siginfo_t status;
@@ -213,7 +213,6 @@ int main(int argc, char *argv[]) {
         cmdline[i++] = "/sbin/fsck";
         cmdline[i++] = "-a";
         cmdline[i++] = "-T";
-        cmdline[i++] = "-l";
 
         if (!root_directory)
                 cmdline[i++] = "-M";
-- 
1.7.3.2

